IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeoScienceObservation/DocEntity.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeoScienceObservation" UniqueId="0af6c427-6c7b-42e9-8cb4-421827c75e4f" BaseDefinition="IfcObservation" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="0326fd07-5fbc-4f6e-8933-636ef84205ec" DefinedType="IfcGeoScienceObservationTypeEnum" AttributeFlags="1">
```

Comment:
Should look into entities without IfcTypeObject comments.

Reply:
Done. (WR=>HasObjectType + PredefinedType=>mandatory)